<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="select_video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pause">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Play/Pause</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="player" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>621</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sld_duration">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="duration">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>600</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>     --/--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capture_start">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>311</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Capture Start Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capture_stop">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>311</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Capture Stop Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="start_time">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>350</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>          --/--</string>
    </property>
   </widget>
   <widget class="QLabel" name="stop_time">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>500</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>          --/--</string>
    </property>
   </widget>
   <widget class="QLabel" name="video_path">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="export_stack">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>650</y>
      <width>311</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export Image Stack</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>660</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
